docs/dependency-updates.md

```diff
@@ -0,0 +1,67 @@
+   - Publish a docker image tagged with the incremented version number to ECR for both Flexion and USTC accounts with the command: `export DESTINATION_TAG=[INSERT NEW DOCKER IMAGE VERSION] && npm run deploy:ci-image`
+   - Deploy as normal by triggering a CircleCI workflow 
+     
+     > Refer to [ci-cd.md](ci-cd.md#docker) for more info on this as needed
+     
+1. PDF Testing
+   1. Check `./Dockerfile-pdf-testing` for our versions of the following scripts:
+      1. `ghostscript`: check to make sure support has not fallen off; our version should be listed on the [Debian Package Tracker](https://tracker.debian.org/pkg/ghostscript) on the _Versions_ table
+      1. `graphicsmagick`: check to make sure support has not fallen off; our version should be listed on the [Debian Package Tracker](https://tracker.debian.org/pkg/graphicsmagick) on the _Versions_ table
+   1. If support for either package has fallen off, we need to update
+      1. Replace the version in the Dockerfile with one of the listed stable versions
+      1. Verify the PDF's still pass by running the commands listed on `./docs/testing.md` under the _PDF Testing_ heading
+1. Validate updates by deploying, with a [migration](./additional-resources/blue-green-migration.md#manual-migration-steps), to an experimental environment. This helps us verify that the package updates don't affect the migration workflow.
+
+### Caveats
+
+Below is a list of dependencies that are locked down due to known issues with security, integration problems within DAWSON, etc. Try to update these items but please be aware of the issue that's documented and ensure it's been resolved.
+
+#### axios
+
+`axios` has a major update available to ^1.0.0 but there are breaking changes and no associated upgrade guide as of writing this. See [this issue](https://github.com/axios/axios/issues/5014). Seems like as of now, there are still quite a few issues popping up with this major update so it may be worthwhile to wait a few days until those are ironed out.
+
+#### puppeteer / puppeteer-core
+
+`puppeteer` and `puppeteer-core` have a major version update to ^18.x.x, but they need to stay at the same major version as `chrome-aws-lambda` (17.1.3). If we upgrade `puppeteer`, we see a ` cannot read property 'prototype' of undefined` error. 
+
+#### s3rver
+
+Check if there are updates to `s3rver` above version [3.7.1](https://www.npmjs.com/package/s3rver).
+- Why is there a patch called `s3rver+3.7.1.patch`?
+    - To address the high severity issue exposed by `s3rver`'s dependency on `busboy` 0.3.1, which relies on `dicer` that actually has the [security issue](https://github.com/advisories/GHSA-wm7h-9275-46v2). Unfortunately, `busboy` ^0.3.1 is incompatible with s3rver which is why there's a patch in place to make it compatible.
+- How does the patch run?
+    - This runs as part of the `npm postinstall` step.
+
+### Incrementing the Node Cache Version
```

Comment:
@rachelschneiderman could you help me clarify what the purpose is of doing this?

Reply:
Sorry, I did not see the notification you had tagged me!

Here are my thoughts:

It's rare to need to increment or change the cache key. One reason you may need/want to do so is if something happens while storing the cache which corrupts it. For example, a few months ago a package failed to install while the cache was being stored. Circle had no idea that the installation didn't go according to plan and saved the corrupted cache. In this case, we  incremented the cache key version so that Circle was forced to reinstall the node dependencies and save them under the new key.  The cache key can be updated by searching within `config.yml` for vX-npm and vX-cypress where X is the current version of the cache key, then increment the version found.

Reply:
Thanks Rachel! This is very helpful.